graphql_api/types/bundle_analysis_report/bundle_analysis_report.py

```diff
@@ -0,0 +1,46 @@
+from ariadne import ObjectType, UnionType
+
+from graphql_api.types.comparison.comparison import MissingHeadReport
+from services.bundle_analysis import BundleAnalysisReport, BundleReport
+
+bundle_analysis_report_result_bindable = UnionType("BundleAnalysisReportResult")
+bundle_analysis_report_bindable = ObjectType("BundleAnalysisReport")
+bundle_report_bindable = ObjectType("BundleReport")
+
+
+@bundle_analysis_report_result_bindable.type_resolver
+def resolve_bundle_analysis_report_result_type(obj, *_):
+    if isinstance(obj, BundleAnalysisReport):
+        return "BundleAnalysisReport"
+    elif isinstance(obj, MissingHeadReport):
+        return "MissingHeadReport"
+
+
+@bundle_analysis_report_bindable.field("sizeTotal")
+def resolve_size_total(bundles_analysis_report: BundleAnalysisReport, info):
+    return bundles_analysis_report.size_total
+
+
+@bundle_analysis_report_bindable.field("loadTimeTotal")
+def resolve_load_time_total(bundles_analysis_report: BundleAnalysisReport, info):
+    return bundles_analysis_report.load_time_total
+
+
+@bundle_analysis_report_bindable.field("bundles")
+def resolve_bundles(bundles_analysis_report: BundleAnalysisReport, info):
+    return bundles_analysis_report.bundles
+
+
+@bundle_report_bindable.field("name")
```

Comment:
Wdyt about separating bundle analysis reports and bundles into their own folders? Rn they're similar but we could eventually make that difference more pronounced, so might make sense to separate that into different folders

Reply:
I don't see there being an entry point into bundles without going through bundle analysis though. If/when that does come up in the future, we can separate it, but for now I think the bundle analysis report and bundle analysis comparison folders separation is intuitive.